### PR TITLE
Fix profile picture issue when using non square images

### LINKED ISSUE #126: [PE-D] Picture renders very oddly
I think the screenshot speaks for itself. Images successfully loads to the correct person but renders wrongly and not in the expected circle shape.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: sheehui/ped#12

### PR DESCRIPTION
**Pull request by Zacchua**
When using very tall and narrow images, image will not be centered in the circle and the circle width would be shorter than normal. 

Fixed this by manually setting x and y points for the image and enclosing the imageview in a hbox of fixed min and max width 

Closes #126

### CODE REVIEW

File: src/main/java/socialite/ui/PersonCard.java [hunk 1]
```diff
@@ -155,6 +155,32 @@ private void makeRemark(Remark remark) {
         }
     }
 
+    // credits for this method goes to https://stackoverflow.com/questions/32781362/centering-an-image-in-an-imageview
+    private void centerImage(ImageView imageView) {
+        Image img = imageView.getImage();
+        if (img != null) {
```
Comment (stanley-1) [inline src/main/java/socialite/ui/PersonCard.java:161]: Great solution! One minor way that you can consider improving it from a Code Quality POV: Use guard clause for the "unusual case" instead. 

E.g.: Deal with `if (img == null) { return; }` which should make the "happy path" prominent as a large chunk of the code [lines 162-179] will no longer be indented by one level.

**Review by stanley-1 — APPROVED** [review]
Good job on the methodical solution @Zacchua! Very neat and organized method too! LGTM!

**Review by david-eom — APPROVED** [review]
Tested on my side, everything works as expected. Thanks for the fix Zac!

File: src/main/java/socialite/ui/PersonCard.java (hunk 1 above)
Comment (Zacchua) [reply src/main/java/socialite/ui/PersonCard.java]: Sorry for the late reply, I missed this out! Fixed in the latest commit!

**Pull request merged**